bin/wasm-node/javascript/src/index.test-d.ts

```diff
@@ -1,15 +1,19 @@
-import { expectType } from 'tsd';
-import smoldot from 'smoldot';
+import smoldot from 'smoldot'; // smoldot is Smoldot
```

Comment:
What does `smoldot is Smoldot` mean?

Reply:
It is a type assertion that means:

 "Is the variable `smoldot` that is returned by the import statement of the type `Smoldot`?"  where the type `Smoldot` is the [type of the default export of the package](https://github.com/paritytech/smoldot/blob/main/bin/wasm-node/javascript/src/index.d.ts#L41-L47)

Reply:
I.E. when `dtslint` parses `index.test-d.ts` it checks that the AST of the type that gets imported from the javascript file matches the interface defined as the export in the typescript typings file.  I missed an explicit test for that before.  Now we will get errors if we add to the public API but don't update the typing file

Reply:
Oh, so this comment is actually interpreted by `dtslint`?